src/js/components/search/filters/awardType/AwardType.jsx

```diff
@@ -39,19 +40,32 @@ const defaultProps = {
 };
 
 const propTypes = {
-    awardTypes: React.PropTypes.arrayOf(React.PropTypes.object)
+    awardTypes: React.PropTypes.arrayOf(React.PropTypes.object),
+    toggleFilter: React.PropTypes.func,
+    hideArrow: React.PropTypes.bool,
+    arrowState: React.PropTypes.string,
+    showFilter: React.PropTypes.bool
 };
 
 export default class AwardType extends React.Component {
 
     render() {
-        const awardTypes = this.props.awardTypes.map((type, index) =>
-            <PrimaryAwardType {...type} {...this.props} key={index} />
-        );
+        let awardTypes = null;
+        if (this.props.showFilter === true) {
+            awardTypes = (
+                this.props.awardTypes.map((type, index) =>
+                    <PrimaryAwardType {...type} {...this.props} key={index} />
+            ));
+        }
+
 
         return (
-            <div className="award-type-filter">
-                <b>Award Type</b>
+            <div className="award-type-filter search-filter">
+                <FilterExpandButton
```

Comment:
Why are the `FilterExpandButton` components in every filter component? Shouldn't these, plus the title of the filter, be in the parent component?